src/CamelKNodeProvider.ts

```diff
@@ -196,7 +196,8 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 			let o = JSON.parse(temp);
 			for (var i=0; i<o.items.length;i++) {
 				var integrationName = o.items[i].metadata.name;
-				let newNode = new TreeNode("string", integrationName, 'Running', vscode.TreeItemCollapsibleState.None);
+				var status = o.items[i].status.phase;
```

Comment:
what do you think to try to provide more explicit naming to `o` and `items[i]`?

I understood that after looking to the context but i'm not even sure.
`o` is the list of integrations represented as JSon?
`o.items[i]` is the Json for one integration?

Reply:
Improved